packages/material-ui/src/Portal/Portal.js

```diff
@@ -69,10 +69,6 @@ Portal.propTypes = {
   onRendered: PropTypes.func,
 };
 
-Portal.defaultProps = {
```

Comment:
Move some of the `.defaultProps` to prop destructuring. I haven't migrated everything. I have left the complex objects and the one used in the custom prop-types.